modules/stats/stats.py

```diff
@@ -36,6 +36,22 @@ def calculate_best_worst_trade(closed_trades):
     return best_trade_ratio, worst_trade_ratio
 
 
+def find_total_value(value: dict, budget: dict, tick: int) -> float:
+    """
+    Finds the total value of given dictionaries at given tick time
+    """
+    total_value = 0
+    try:
+        total_value += value[tick]
+    except KeyError:
+        pass
+    try:
+        total_value += budget[tick]
+    except KeyError:
+        pass
+    return total_value
```

Comment:
The purpose of the variable name `value` is a bit hard to grasp in this context. Maybe `open_order_value`?

I personally like this a bit better. Noticed that you just moved it BTW
```suggestion
def get_total_value_at_tick(value: dict, budget: dict, tick: int) -> float:
    return value.get(tick, 0) + budget.get(tick, 0)
```

Reply:
Done in latest commit!